script/int-log-compiler.sh

```diff
@@ -171,7 +190,16 @@ for i in $(seq ${BACKOFF_MAX}); do
     echo "Starting simulator on port ${SIM_PORT_DATA}"
     simulator_start ${SIM_BIN} ${SIM_PORT_DATA} ${SIM_LOG_FILE} ${SIM_PID_FILE} ${SIM_TMP_DIR}
     sleep 1 # give daemon time to bind to ports
+    if [ ! -s ${SIM_PID_FILE} ] ; then
+	echo "Simulator PID file is empty or missing. Giving up."
+	exit 1
```

Comment:
More tabs. Again, please read the coding standard: https://github.com/tpm2-software/tpm2-tss/blob/master/doc/coding_standard_c.md#whitespace

Reply:
I'll fix the tabs. Yes I read the coding standard--it's just a habit :-)